content/src/thumbnail.js

```diff
@@ -61,12 +78,13 @@ function getImageInfo(canvas) {
 
   // Find the longer edge and make it a square.
   var longerEdge;
+  var diff = Math.abs((imageWidth - imageHeight) / 2);
```

Comment:
Yes it should never grow. I would change that. 

Previously if there is no output there will be no canvas, then it will not do the capturing. However, using `html2canvas`, there will always be a result canvas even if it is blank. I would try to figure out a good way to do this.